opsimulator/opsimulator.go

```diff
@@ -468,6 +475,22 @@ func (opSim *OpSimulator) checkInteropInvariants(ctx context.Context, logs []typ
 				return fmt.Errorf("executing and initiating message fields are not equal")
 			}
 
+			interopStart, err := opSim.crossL2Inbox.InteropStart(&bind.CallOpts{})
+			if err != nil {
+				return fmt.Errorf("failed to fetch interop start: %w", err)
+			}
+			if interopStart.Cmp(identifier.Timestamp) > 0 {
```

Comment:
replaces this check being removed from CrossL2Inbox: https://github.com/ethereum-optimism/optimism/pull/13626/files#diff-c899f14eedc7928371b7e497dd12658c2452d9a171856b2b001cc20885fe6dc8L192